queries/rust/textobjects.scm

```diff
@@ -47,26 +47,13 @@
 
 (match_expression) @conditional.outer
 
-(if_let_expression
-  consequence: (block)?
```

Comment:
I'm not sure how much it matters, but in the `if_expression` equivalent the `(block)` isn't optional as it is in this case.

Reply:
I don't think rust even allows optional block in `if` expression.